core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -38,7 +38,13 @@ object ConsoleConsumer extends Logging {
 
   def main(args: Array[String]) {
     val conf = new ConsumerConfig(args)
-    run(conf)
+    try {
+      run(conf)
+    } catch {
+      case e: Throwable =>
+        error("Error running consumer: ", e)
+    }
+    System.exit(0);
```

Comment:
I am not sure if I should remove the brace around `try`. Is this the recommended style for scala code in Kafka? I am asking because braces are added for single expression `try` in most parts of Kafka code, e.g. SocketServerTest.scala, Log.scala, ZkUtils.scala. I find it a bit confusing if `catch` has braces while try doesn't -- it only happened once in ConsoleConsumer.scala. What do you think?

Also, how about I use `System.exit(1)`? I find that most Kafka tools use `1` as error code. MirrorMaker is the only case which does `System.exit(-1)`.